viser/_messages.py

```diff
@@ -308,3 +308,5 @@ class ThemeConfigurationMessage(Message):
     """Message from server->client to configure parts of the GUI."""
 
     canvas_background_color: int
+    show_titlebar: bool
+    titlebar_content: str
```

Comment:
do you think it's possible to type this more specifically? it's a little bit scary to just say the titlebar_content can be any string!

maybe the `TitlebarButton`, `TitlebarImage` types could be used directly here, and translated automatically to the typescript definition

(we could, as an example, modify the typescript interface gen script to add support for `TypedDict`: https://github.com/nerfstudio-project/viser/blob/main/viser/infra/_typescript_interface_gen.py)

Reply:
maybe we can break it into three fields, like:
- `titlebar_left: Tuple[TitlebarObject, ...]`
- `titlebar_center: Tuple[TitlebarObject, ...]`
- `titlebar_right: Tuple[TitlebarObject, ...]`